youngmasterhiei/p1react
# consolidate the api

# Things Done
- consolidate all API calls from the `client` into a single place
- inject `REACT_APP_API_LOCATION` and `REACT_APP_API_PORT` at build time
  - use these when hitting the API
- rip out webpack
  - greatly simplifies things
  - don't use webpack to manually configure stuff unless
    - you know what you're doing (gonna have a lot of  strange issues otherwise)
    - you absolutely need to finetune the build process in a different direction
- ran prettier everywhere
- general cleanup

# How To Test
- please just click around, ensure no regressions. no idea what the state of this thing was before i did this.
